ote_sdk/ote_sdk/test_suite/training_tests_actions.py

```diff
@@ -490,6 +542,10 @@ def _run_ote_nncf(
         assert (
             self.nncf_model.model_format == ModelFormat.BASE_FRAMEWORK
         ), "Wrong model format"
+        assert check_nncf_model_graph(
```

Comment:
@AlexanderDokuchaev Could you create a separate test to check the compression graph?